tests/integration/210.command.deploy.test.js

```diff
@@ -124,17 +124,12 @@ if (process.env.NETLIFY_TEST_DISABLE_LIVE !== 'true') {
         .withNetlifyToml({
           config: {
             build: { publish: 'public', command: 'echo "no op"' },
-            edge_functions: [{ function: 'yell', path: '/*' }],
+            edge_functions: [{ function: 'edge', path: '/*' }],
           },
         })
         .withEdgeFunction({
-          handler: async (_, context) => {
-            const resp = await context.next()
-            const text = await resp.text()
-
-            return new Response(text.toUpperCase(), resp)
-          },
-          name: 'yell',
+          handler: async () => new Response('Edge Function works'),
```

Comment:
Not a blocker at all, but I'm curious about why you changed the test here. The previous test edge function was a bit more comprehensive, in that it was calling the origin and making a transformation, whereas the new one is just delivering a response.

Reply:
Isn't that what we are testing here? That the function was deployed correctly and responded?

Reply:
I guess I can revert it, but I thought this is simpler and does exactly what we are trying to test here. And I guess/hope we have some tests somewhere that asserts that the edge internals work.

Reply:
> I can revert it if you want, but I thought this is simpler and does exactly what we are trying to test here. And I guess/hope we have some tests somewhere that asserts that the edge internals work.

No need to revert.